### PR TITLE
hardware/controllers: Add "versionadded" and some minor fixes

### PR DESCRIPTION
**Pull request by Holzhaus**

### CODE REVIEW

File: source/hardware/controllers/pioneer_ddj_sb.rst [hunk 1]
```diff
@@ -15,8 +17,11 @@ Windows
 
 Windows Vista, Windows 7 and Windows 8 are supported. You can download the latest drivers and firmware from http://www.pioneerdj.com/en/support/software/ddj-sb.
 
-**IMPORTANT for Windows users**: If you are having issues getting both sound outputs to work properly, please try using a different Sound API (see the Preferences menu). DirectSound seems to have
-issues with this controller. WASAPI seems to work fine, as does MME, although only with higher latencies. On Windows 10, only MME works.
+.. note::
+   If you're using windows and have problems getting both sound outputs to work properly, please try using a different Sound API (see the Preferences menu).
```
Comment (ronso0) [inline source/hardware/controllers/pioneer_ddj_sb.rst:21]: reference API?
Add a link to Pref > Sound hardware?

**Comment by ronso0** [comment]
LGTM!

File: source/hardware/controllers/pioneer_ddj_sb.rst (hunk 1 above)
Comment (Holzhaus) [reply source/hardware/controllers/pioneer_ddj_sb.rst]: Done.

**Comment by ronso0** [comment]
still LGTM :)
do you consider this finished?

btw some forum links point to the old forum.
checking and repairing this and other links is too much for this PR, so I filed an issue
https://github.com/mixxxdj/manual/issues/223

**Comment by Holzhaus** [comment]
> do you consider this finished?

Yes. Feel free to merge. When #151 has been merged, I'll probably post a new PR that adds `:hwlabel:` roles that we can use for button/knob labels on controllers (similar to `:guilabel:`), to improve the readability of the hardware manuals.

**Pull request merged**